internal/pkg/hub/manifest.go

```diff
@@ -0,0 +1,105 @@
+package hub
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httputil"
+
+	"github.com/confluentinc/cli/internal/pkg/cpstructs"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/log"
+)
+
+const clientNotInitializedErrorMsg = "Hub client not initialized"
```

Comment:
Do we still want to refer to this as "Confluent Hub" or are we trying to deprecate that terminology?

Reply:
The website for plugins still calls it Confluent Hub, so I want to be consistent with it.